float32 x_pos
float32 y_pos
int32 target_id
float32 effectiveness
int32 confidence_index
float32 attrition_estimate
bool ready


